source/process/overview.rst

```diff
@@ -0,0 +1,121 @@
+Development Process Overview
+============================
+
+This document describes the process through which feedback and design discussions flow into community systems, then into tickets, then into pull requests, then into bi-monthly releases based on product purpose.
+
+Purpose
+-------
+
+The core offer for users of Mattermost is: **All your team communication in one place, searchable and accessible anywhere.**
+
+The design is successful if 100% of team members use Mattermost for internal communications, and are largely off of email and proprietary SaaS products that lock-in user data as part of their business model.
+
+See `Mattermost Manifesto <https://www.mattermost.org/manifesto/>`__ for more details.
+
+Community Systems
+-----------------
+
+Please see `Community Systems <http://docs.mattermost.com/process/community-systems.html>`__ documentation for how feedback from community constituents is processed into `tickets accepting pull requests. <http://docs.mattermost.com/process/accepting-pull-request.html>`__
+
+Tickets
+-------
+
+Mattermost priorities are managed in Jira tickets, which are created by the core team via feedback from community systems as well as through the planning processes.
+
+Triage
+~~~~~~
+
+On non-holiday weekdays new tickets are reviewed in a process called "triage", and assigned a Fix Version of "backlog", indicating the ticket has enough specificity that it can be assigned to a developer to be completed. If the ticket is a bug or an improvement, it is assigned a Fix Version of the next release.
```

Comment:
I don't think we should be putting all improvements in the next release. Bugs should have fix version of the current release. 

Also I believe we have "unscheduled" fix version instead of backlog now.